### PR TITLE
Basic description for Fault Injection Simulator (FIS).

### PR DESCRIPTION
**Pull request by taras-kobernyk-localstack**

### CODE REVIEW

File: content/en/aws/fis/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "FIS"
+linkTitle: "FIS"
+categories: ["LocalStack Pro"]
+description: >
+  Fault Injection Simulator (FIS)
+---
+
+Basic support for FIS is included in LocalStack Pro. The local AppSync API allows you to introduce faults to other services - in order to check how your setup behaves when parts of it stop working.
```
Comment (whummer) [inline content/en/aws/fis/index.md:9]:
```suggestion
Basic support for the Fault Injection Simulator (FIS) service is included in LocalStack Pro. The local FIS API allows you to introduce faults to other services - in order to check how your setup behaves when parts of it stop working.
```

File: content/en/aws/fis/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+The full list of such possible fault introductions - called "actions" - available in AWS is [here](https://docs.aws.amazon.com/fis/latest/userguide/fis-actions-reference.html).
```
Comment (whummer) [inline content/en/aws/fis/index.md:10]:
```suggestion
The full list of such possible fault injections - called "actions" - is available in the [AWS docs](https://docs.aws.amazon.com/fis/latest/userguide/fis-actions-reference.html).
```

File: content/en/aws/fis/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+In general, calls to FIS contain the following information:
+
+1. What kind of fault to introduce - "action".
+2. What kind of resources to affect - "target".
+3. (sometimes) Duration of the interruption. After the specified amount of time FIS is supposed to revert systems to their original state or to stop introducing faults.
+
+FIS actions come in roughly two types:
+
+1. A single-time event, e.g. aws:ec2:stop-instances FIS action is a stop-instances command sent to target EC2 instances. Results of some of these single-time events can be automatically reverted after a specified period of time, e.g. by sending start-instances command to the affected instances.
```
Comment (whummer) [inline content/en/aws/fis/index.md:20]:
```suggestion
1. A single-time event, e.g., `aws:ec2:stop-instances` FIS action is a `stop-instances` command sent to target EC2 instances. Results of some of these single-time events can be automatically reverted after a specified period of time, e.g., by sending `start-instances` command to the affected instances.
```

File: content/en/aws/fis/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+2. Generation of API errors in response to a specified percentage of API calls. E.g. aws:fis:inject-api-unavailable-error. Currently AWS only supports this for EC2 API calls.
```
Comment (whummer) [inline content/en/aws/fis/index.md:21]:
```suggestion
2. Generation of API errors in response to a specified percentage of API calls, e.g., `aws:fis:inject-api-unavailable-error` to inject an HTTP 503 error. Currently AWS only supports this for EC2 API calls.
```

File: content/en/aws/fis/index.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+# Most notable current FIS limitations in LocalStack
+1. Only some actions, available in AWS, are available in LocalStack. Others generate an error. The list of available actions can be easily extended on demand.
```
Comment (whummer) [inline content/en/aws/fis/index.md:24]:
```suggestion
1. Only a subset of FIS actions available in AWS are currently supported in LocalStack (unsupported actions generate an error). The set of supported actions is extended on an ongoing basis, and new actions can easily be added on demand.
```

File: content/en/aws/fis/index.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+2. Only single-time events are supported. The continuous generation of API errors in reponse to some API calls is in the works.
+3. LocalStack doesn't support target selection mechanism used by AWS. See [selection mode documentation for more info](https://docs.aws.amazon.com/fis/latest/userguide/targets.html#target-selection-mode)
+4. LocalStack ignores [roleArns](https://docs.aws.amazon.com/fis/latest/APIReference/API_ExperimentTemplate.html#fis-Type-ExperimentTemplate-roleArn). In AWS FIS performs actions under permissions granted to given roleArns.
```
Comment (whummer) [inline content/en/aws/fis/index.md:27]:
```suggestion
4. LocalStack currently ignores [`roleArn`s](https://docs.aws.amazon.com/fis/latest/APIReference/API_ExperimentTemplate.html#fis-Type-ExperimentTemplate-roleArn). In AWS FIS performs actions under permissions granted to given `roleArn`s.
```

Comment (whummer) [reply content/en/aws/fis/index.md:27]: Note: we may be able to integrate this with our new IAM enforcement mechanism in the near future.. 👍

**Review by whummer — APPROVED** [review]
Awesome, great to see this added @taras-kobernyk-localstack ! 🚀 Added a few small comments, mostly minor nitpick suggestions.. 

Would be great to get a quick feedback from @HarshCasper as well.. 👍

**Review by HarshCasper — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**